### PR TITLE
style button badge with css

### LINKED ISSUE #1165: Tooltip color change on "Test Pilot" button after enabling "Tab Center" experiment
[Affected versions]:
- Nightly 50.0a1
- Test Pilot 0.8.3 -tag-2016-07-29

[Affected Platforms]:
- All Windows
- All Mac OS
- All Linux

[Steps to reproduce]:
1. Go to https://testpilot.stage.mozaws.net page.
2. Click on the big green "Install the Test Pilot Add-on" button.
3. Observe the "Test Pilot" button from browser toolbar.
4. Chose the "Tab Center" experiment and "Enable" it.
5. Observe again the "Test Pilot" button from browser toolbar.

[Expected result]:
In both cases the "Test Pilot" button is displayed with a blue "New" tooltip over it.

[Actual result]:
Step 3 - the "Test Pilot" button is displayed with a blue "New" tooltip over it.
Step 5 - after "Tab Center" installation, the tooltip changes its color into red. 

[Notes]:
- Added a screen recording with the issue https://goo.gl/uCdqbH.

### PR DESCRIPTION
**Pull request by dannycoates**
fixes #1165 

Thanks @bwinton for the css tip and technique 🍻

### CODE REVIEW

**Comment by dannycoates** [comment]
If this looks good I've got a follow-up commit to use css for the icon color instead of the js shenanigans.

File: addon/lib/toolbar-button.js [hunk 1]
```diff
@@ -8,17 +8,20 @@ const {Panel} = require('sdk/panel');
...
 const tabs = require('sdk/tabs');
+const self = require('sdk/self');
 
 const Mustache = require('mustache');
 const templates = require('./templates');
 Mustache.parse(templates.installed);
 Mustache.parse(templates.experimentList);
 
+const xulcss = require('./xulcss');
+xulcss.addXULStylesheet(self.data.url('button.css'));
```
Comment (bwinton) [inline addon/lib/toolbar-button.js:19]: Drive-by review: You probably want to remove the stylesheet on disable/uninstall, too.  It wouldn't _really_ hurt anything to leave it added, but it's nice to clean up after yourself.  😉

Comment (dannycoates) [inline addon/lib/toolbar-button.js:19]: Doesn't it do this automatically here? https://github.com/dannycoates/testpilot/blob/85d9fd2be75aa4264497843265a2ca74fc73e9d1/addon/lib/xulcss.js#L42

Comment (bwinton) [inline addon/lib/toolbar-button.js:19]: Well huh!  TIL.  😃
/me goes to remove a bunch of cleanup code from Whimsy…  😉

**Comment by lmorchard** [comment]
Works for me!
